app/views/admin/events/_event_form.html.erb

```diff
@@ -1,50 +1,54 @@
-<div class="form-group">
-  <%= f.label :cover_image %>:
-  <%= f.file_field :cover_image, class: "form-control" %>
+<div class="crayons-card p-6">
+  <%= form_for [:admin, @event] do |f| %>
+    <div class="form-group">
+      <%= f.label :cover_image %>:
+      <%= f.file_field :cover_image, class: "form-control" %>
+    </div>
+    <div class="form-group">
+      <%= f.label :profile_image %> (for live notification):
+      <%= f.file_field :profile_image, class: "form-control" %>
+      <img src="<%= @event.profile_image_url %>" style="width: 25%;" alt="event profile image">
+    </div>
+    <div class="form-group">
+      <%= f.label :title %>
+      <%= f.text_field :title, maxlength: 90, size: 40, required: true, class: "form-control" %>
+    </div>
```

Comment:
For future (well, you can also do it here if you want to) we could reuse Crayons markup for form fields... So this one in particular would look as follows:

```suggestion
<div class="crayons-field">
  <%= f.label :title, class: "crayons-field__label" %>
  <%= f.text_field :title, maxlength: 90, size: 40, required: true, class: "crayons-textfield" %>
</div>
```

^ and that applies to all form fields.